src/views/ont/OntReception.vue

```diff
@@ -27,50 +31,53 @@ import Alert from '@/components/Alert'
 import Helper from '@/mixins/Helper'
 import { getPlates, transformPlates} from '@/api/SequencescapePlates'
 import CREATE_PLATE_WITH_SAMPLES from '@/graphql/queries/CreatePlateWithSamples.mutation.gql'
+import Spinner from 'vue-simple-spinner'
+import Api from '@/mixins/Api'
 
 export default {
   name: 'Reception',
-  mixins: [Helper],
-  props: {
-  },
+  mixins: [Helper, Api],
   data () {
     return {
       barcodes: [],
-      plates: {}
+      busy: false
     }
   },
   components: {
-    Alert
+    Alert,
+    Spinner
   },
   methods: {
-    async getSequencescapePlates (barcodes) {
-      return await getPlates(this.$store.getters.api.sequencescape.plates, barcodes)
-    },
-    getBarcodes () {
-      return this.barcodes.split('\n').filter(Boolean).join(',')
+    async getSequencescapePlates() {
+      await getPlates(this.sequencescapeRequest, this.formattedBarcodes)
```

Comment:
I think this needs a return

Reply:
Oh interesting thanks, I couldnt test with valid barcodes becuase the plate I was generating didnt have tagged samples, so might need you to test again after changes! But ill add that return :)

Reply:
Again could we add a test for this?

Reply:
Oh Its a method, not a computed, so dont think it needs the return?

Reply:
Ah I remember trying, but struggled as getPlates is being imported in from the SequencescapePlates api, so couldnt mock it.

Reply:
Ah I remember now trying, but couldnt as getPlates is imported so struggled to mock it